--- rpm-4.17.0/lib/rpmds.c~	2021-06-21 12:00:44.000000000 +0000
+++ rpm-4.17.0/lib/rpmds.c	2022-08-08 22:18:36.487254362 +0000
@@ -1139,11 +1139,9 @@
     { "rpmlib(DynamicBuildRequires)", "4.15.0-1",
 	(RPMSENSE_RPMLIB|RPMSENSE_EQUAL),
     N_("support for dynamic buildrequires.") },
-#ifdef HAVE_ZSTD
     { "rpmlib(PayloadIsZstd)",		"5.4.18-1",
 	(RPMSENSE_RPMLIB|RPMSENSE_EQUAL),
     N_("package payload can be compressed using zstd.") },
-#endif
     { NULL,				NULL, 0,	NULL }
 };
 
